src/Annotation/Breadcrumb.php

```diff
@@ -20,12 +20,12 @@ class Breadcrumb
     /**
      * @var string Title of the breadcrumb
      */
-    private $title = null;
```

Comment:
Is the removals of `null` a result of the `nullable_type_declaration_for_default_null_value` PHP CS Fixer change? Not having them set to `null` could after construction potentially lead to exceptions on read when not being instantiated.